roles/glance/tasks/main.yml

```diff
@@ -86,3 +86,9 @@
     - logrotate
     - logging
   when: logging.enabled|default('True')|bool
+- name: change glance log file permissions
+  file: path={{ item }} mode=0644 owner=glance group=glance state=file
```

Comment:
Just like with keystone, the state should be touch, not file, so that permissions get in place on files that don't exist yet.